common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -282,7 +289,7 @@ div.video {
         }
       }
 
-      ul.vcr {
+      div.vcr {
```

Comment:
I just noticed that .vcr seems to be a wrapper class around .vidtime. If that is its primary use, I could consider using ".wrapper-video-duration" instead of "vcr" and ".video-duration" instead of .vidtime